core/src/lib.rs

```diff
@@ -10,10 +10,16 @@
 
 extern crate edn;
 extern crate ordered_float;
+#[macro_use]
+extern crate lazy_static;
```

Comment:
Nit: newline above `#[macro_use]`, move this above `ordered_float`, and shuffle `edn` down to here. We do this order:

- extern crate: libs
- extern crate: other mentat modules
- `use` libs
- `use` other mentat modules
- `use` modules in the same crate